modules/crumbtrail/main/index.coffee

```diff
@@ -4,6 +4,8 @@ class Crumbtrail
     hx.component.register(@selector, this)
     self = this
 
+    hx.select(@selector).classed('hx-crumbtrail', true)
```

Comment:
If you stored this as `selection` you could re-use it below where we define the `@view` (https://github.com/alpacaaa/hexagonjs/blob/5d44760a18e4184b208b7e7397da1e01ded7e1c9/modules/crumbtrail/main/index.coffee#L24)